sct.py

```diff
@@ -963,50 +996,62 @@ def send_email(test_id=None, test_status=None, start_time=None, started_by=None,
         with open(os.path.join(testrun_dir, 'test_id'), 'r', encoding='utf-8') as file:
             test_id = file.read().strip()
         email_results_file = os.path.join(testrun_dir, "email_data.json")
+        if not os.path.exists(email_results_file):
+            email_results_file = "email_data.json" if os.path.exists("email_data.json") else None
+        if not email_results_file:
+            LOGGER.error("Results file not found")
+            sys.exit(1)
```

Comment:
Why this code needed?

Reply:
May it is better to save under the same file like for longevity?

Reply:
in case there is no `email_results_file` the script will exit with `RC=1`

Reply:
> May it is better to save under the same file like for longevity?

it is based on the same logic as longevity tests, but because of performance tests run on builders, and not SCT runners, it might differ a bit, but same logic for both cases

Reply:
am i right, that the email_data.json is stored under scylla-cluster-tests/ dir on builder? Let store it under test log dir

Reply:
> am i right, that the email_data.json is stored under scylla-cluster-tests/ dir on builder? Let store it under test log dir

one issue i see (at least on builders) is that for `--logdir` is sent as `--logdir "`pwd`"` so it won't be in the right path anyway